Documentation/build-arm64-firmware.md

```diff
@@ -118,16 +117,19 @@ This document describes how to set up a build environment to build the latest fi
     export TA_CROSS_COMPILE=~/gcc-linaro-7.2.1-2017.11-x86_64_aarch64-linux-gnu/bin/aarch64-linux-gnu-
     export TA_CPU=cortex-a53
     
-    pushd ~/ms-tpm-20-ref/Samples/ARM32-FirmwareTPM/optee_ta
+    pushd ~/MSRSec/TAs/optee_ta
     
-    make
+    make CFG_ARM64_ta_arm64=y CFG_FTPM_USE_WOLF=y CFG_AUTHVARS_USE_WOLF=y
     
```

Comment:
CFG_ARM64_ta_arm64=y shouldn't be necessary. It's already defined in github/optee_os/out/arm-plat-imx/conf.mk

Reply:
This make command doesn't build the fTPM.  The makefile in MSRSec has a bug.  The ftpm target is lowercase along with .phony, but all: has fTPM capitalized.

Reply:
Thanks. I'll take a look at the build target. The final versions have been sitting in pull requests in MSRsec for a while so there might be an inconsistency.

Reply:
Fixed, I'll also update MSRsec to fix the fTPM->ftpm.